src/stdatamodels/jwst/transforms/tests/test_transforms.py

```diff
@@ -560,7 +563,7 @@ def test_nircam_grism_roundtrip(direction):
     xi, yi, wli, ordersi = combined.evaluate(x0, y0, wl, orders)
     assert_allclose(xi, x0)
     assert_allclose(yi, y0)
-    assert_allclose(wli, wl)
+    assert_allclose(wli, wl, rtol=1e-4)
```

Comment:
Prior to these changes, this test was passing by pure numerical chance.  In fact, making the sampling finer would cause the test to fail.  For a 40-point sampling I think a relative difference of 1e-4 is pretty decent.